src/platform/site-wide/sass/modules/_m-crisis-line.scss

```diff
@@ -151,16 +151,6 @@ button.va-crisis-line {
     }
   }
 
-  &-icon {
```

Comment:
This affected the icons in the Crisis Line modal. See screenshot from the review instance

<img width="525" alt="Screenshot 2024-07-25 at 9 39 31 AM" src="https://github.com/user-attachments/assets/463ac252-740f-4384-9dd8-c6691d46b4f7">

Reply:
I have reinstated this code.  Closer inspection revealed that `&-icon` classes also included icons that were inline .svg's;  I am not sure how that relates protocol wise, but it is beyon the scope of a Fontawesome removal.  Am also looking at other PRs for similar patterns now. Cc:@micahchiang

Reply:
@rmessina1010 Yep, when Sitewide removed Font Awesome icons for our products, we were advised to use inline SVGs for everything in the header/footer (because of the injected header/footer).